data-raw/process_data_set_danica.R

```diff
@@ -0,0 +1,64 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
+#Historical Raptor by Player
+historical_raptor_by_player <- read_csv("data-raw/nba-raptor/historical_RAPTOR_by_player.csv") %>%
+  mutate(
+    era = factor(c("historical"))
```

Comment:
Could you do the converting to factor after you've `bind_rows()` the data frame? Otherwise the saved levels are off. Doing it this way yields `era` being a character